application-diagram-api/src/main/java/com/xwiki/diagram/internal/handlers/DiagramMacroSolrMetadataExtractor.java

```diff
@@ -99,4 +146,15 @@ private String transformName(String name)
             return name;
         }
     }
+
+    private boolean isValid(String name)
+    {
+        if (this.entityNameValidationConfiguration.useValidation()
```

Comment:
If we want to transform the names even if the  "Validate names before saving" is disabled I can delete the check for this.entityNameValidationConfiguration.useValidation().